docs/mkdocs/integration/hub-spoke.md

```diff
@@ -0,0 +1,363 @@
+# Hub-Spoke deployment
+
+Using [Open Cluster Management](https://open-cluster-management.io/), allows to configure and define policies in our control cluster, named `hub` that later can be applied to satellite clusters named `spoke`.
+
+We can leverage OCM to deploy KMM on the environment and have the different components orchestrated by OCM.
+
+!!! note "Using podman instead of docker"
+
+    If you're using podman like we'll do, we need to symlink the command to the `docker` name, as the `Makefile` does uses docker.
+
+    `ln -s /usr/bin/podman /usr/bin/docker`
+
+First we need to prepare our registry, if you're using <https://quay.io>, remember to change repositories created to public if those were not created beforehand once you've pushed to them or you won't be able to consume the images.
+
+## Dependencies
+
+- `registry.ci.openshift.org` token - [Documentation](https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-log-in-to-pull-images-that-require-authentication)
+
+!!! warning "`quay.io` account"
+
+    if the repositories are not created beforehand, please make sure to convert them to public ones after pushing to them
+
+## Setup the Hub
+
+1. Build and push the KMM-Hub container image, the KMM-Hub OLM bundle and catalog from the midstream repository:
```

Comment:
Can't we just consume the operator directly from operatorhub.io? Why does a customer will need to build it from source?